tests/check_framework/tests.py

```diff
@@ -302,6 +302,26 @@ class ModelWithDescriptorCalledCheck(models.Model):
         self.assertEqual(errors, expected)
 
 
+class ModelIndexesTests(SimpleTestCase):
```

Comment:
there's also `invalid_models_tests` where the other model checks are tested, this test should be there